TACC/Core-Portal
# Task/TV3-42: Jobs Listing

## Overview
Transition job listing/job details components to Tapis v3.


## Related

* [TV3-42](https://jira.tacc.utexas.edu/browse/TV3-42) [Partially]
* [TV3-66](https://jira.tacc.utexas.edu/browse/TV3-66)

## Changes

## Testing
1. Ensure job listing view works (https://cep.test/workbench/history/jobs). (I had to skip onboarding to test)
2. Ensure the details modal works.
3. Ensure tests run (minus the job post views in the backend)

## UI



## Notes
Removed Job post/submission related changes to avoid collision with Nathan's work on app form.
Thus, some backend unit tests related to that fail..

Commented out `JobSubmission` from that part to pass `flake` linting.

Wasn't sure of an example slurm job details fixture for frontend tests so just have it the same as job for now will work on in future.
